edx_credentials_themes/static/edx.org/certificates/sass/_build.scss

```diff
@@ -0,0 +1,26 @@
+// ------------------------------
+// MIT Professional Education: Shared Build
```

Comment:
Is this file specific to a particular type of cert?

Reply:
Deleted - Its for the base of the certificate themes. Not sure why it mentioned MIT here.